docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/01_LORIS_Install/Troubleshooting.md

```diff
@@ -58,6 +58,21 @@ Active = 'Y'
 Pending_approval = 'N'
 Password_expiry column value is later than today's date
 ```
+If your admin user has been deactivated due to inactivity, you can get past that using the following steps:
+
+1. Enter `mysql` console with a root user and run the following command:
+   ```
+   use lorisdb;
+   ```
+2.  Make sure the `Active` column in the `users` MySQL table is set to `Y` for this user, if not you can do so with the following command:
+       ```
+       UPDATE users SET active = "Y" WHERE UserID = "lorisadmin";
```

Comment:
I believe  the UserID is typically `admin` not `lorisadmin`
(In a typical production setup, `lorisadmin` is a mysql user in production typically, `admin` is the frontend user)